metrics_utility/automation_controller_billing/collectors.py

```diff
@@ -464,3 +480,56 @@ def main_host_table(since, full_path, until, **kwargs):
     return _copy_table(
         table='main_host', query=f'COPY ({query}) TO STDOUT WITH CSV HEADER', path=full_path, prepend_query=yaml_and_json_parsing_functions()
     )
+
+
+@register('total_workers_vcpu', '1.0', format='json', description=_('Total workers vCPU'))
+def total_workers_vcpu(since, full_path, until, **kwargs):
+    if 'total_workers_vcpu' not in get_optional_collectors():
+        return None
+
+    cluster_name = os.environ.get('METRICS_UTILITY_ANSIBLE_SAAS_CLUSTER_NAME')
```

Comment:
I wonder if having "ANSIBLE_SAAS" in here is meaningful? Better to just leave it as `METRICS_UTILITY_CLUSTER_NAME` perhaps? I don't believe other env vars reference the saas.